bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccBindingConstants.java

```diff
@@ -36,9 +36,12 @@ public class EvccBindingConstants {
     public static final String CHANNEL_BATTERY_CAPACITY = "batteryCapacity";
     public static final String CHANNEL_BATTERY_POWER = "batteryPower";
     public static final String CHANNEL_BATTERY_SOC = "batterySoC";
-    public static final String CHANNEL_BATTERY_PRIORITY_SOC = "batteryPrioritySoC";
```

Comment:
As there already are breaking changes, I’m fine with this breaking change as well.